ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-2023.1-main.yaml

```diff
@@ -0,0 +1,83 @@
+binary_build_commands: 'mkdir -p .workspace; echo {\"gitref\": \"$(git rev-parse \
+  --short HEAD)\"} > .workspace/metadata.json'
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+  use_build_cache: true
+images:
+- build_args:
+  - name: NVIDIA
+    value: "1"
+  - name: BASE_IMAGE
+    value: nvcr.io/nvidia/cuda:11.8.0-runtime-ubi8
+  - name: RELEASE_BASE_IMAGE
+    value: nvcr.io/nvidia/cuda:11.8.0-runtime-ubi8
+  - name: BASE_OS
+    value: redhat
+  - name: PROJECT_VERSION
+    value: 2023.1.0
+  - name: ov_source_branch
+    value: releases/2023/1
+  - name: ov_source_org
+    value: openvinotoolkit
+  - name: ov_contrib_branch
+    value: releases/2023/1
+  - name: ov_contrib_org
+    value: openvinotoolkit
+  - name: ov_use_binary
+    value: "0"
+  - name: sentencepiece
+    value: "1"
+  - name: CHECK_COVERAGE
+    value: "1"
+  - name: RUN_TESTS
+    value: "1"
+  - name: debug_bazel_flags
+    value: --strip=always
+  - name: CMAKE_BUILD_TYPE
+    value: Release
+  - name: GPU
+    value: "1"
+  dockerfile_path: Dockerfile.redhat
+  to: model-server-main-cuda-intel-gpu
+promotion:
+  name: model-server-main-cuda-intel-gpu
+  namespace: opendatahub-io
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+  model-server-main-cuda-intel-gpu:
+    limits:
+      cpu: "12"
+      memory: 24Gi
+    requests:
+      cpu: "6"
+      memory: 12Gi
+tests:
+- as: pr-image-mirror
+  run_if_changed: ^(Dockerfile.redhat)
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: model-server-main-cuda-intel-gpu
+    env:
+      IMAGE_REPO: openvino_model_server
+    workflow: opendatahub-io-ci-image-mirror
+- as: fast-image-mirror
+  cron: 0 0 * * 0
```

Comment:
It's looking like the cron job conflicts with the postsubmit job. Let's delete this cron line and add a third task of the form:

```
- as: nightly-image-mirror
  cron: 0 0 * * 0 ?
  steps:
    dependencies:
      SOURCE_IMAGE_REF: caikit-tgis-serving
    env:
      IMAGE_REPO: caikit-tgis-serving
      RELEASE_VERSION: rhods-1.31
    workflow: red-hat-data-services-ci-image-mirror
```

Likely needs an update to the same values here, etc. Would need for both release and main